src/TCommandLine.cpp

```diff
@@ -439,20 +465,19 @@ bool TCommandLine::event(QEvent* event)
                 ke->accept();
                 return true;
 
-            }
-
-            // Process as a possible key binding if there are ANY modifiers,
-            if (processPotentialKeyBinding(ke)) {
+            } else if (keybindingMatched(ke)) {
+                // Process as a possible key binding if there are ANY modifiers,
                 return true;
-
+            } else {
+                break;
             }
 
             processNormalKey(event);
```

Comment:
This looks like dead code now :frowning:

Reply:
Are you sure - it's intended to quit the switch and call `return QPlainTextEdit::event(event);`

Reply:
This is the whole case now:
```cpp
        case Qt::Key_C:
            if (((ke->modifiers() & allModifiers) == Qt::ControlModifier)
                && (mpConsole->mUpperPane->mSelectedRegion != QRegion(0, 0, 0, 0))) {

                // Only process as a Control-C if it is EXACTLY those two keys
                // and no other AND there is a selection active in the TConsole
                mpConsole->mUpperPane->slot_copySelectionToClipboard();
                ke->accept();
                return true;

            } else if (keybindingMatched(ke)) {
                // Process as a possible key binding if there are ANY modifiers,
                return true;
            } else {
                break;
            }

            processNormalKey(event);
            return false;
```

Every part of the if cascade as a `return` or `break` in it, which means (according to my understanding), that there is no way to reach `processNormalKey(event);` in this case. Because if an `if` is triggered, we return, if we fall through to the `else`, the `break` is reached and we end the switch.

Reply:
Aha. I was thinking of a different case where this is applied. Fixed this now to be like the previous behaviour here. Not sure _why_ do we have it such for the `C` key - perhaps for `Ctrl+C` to work.

Reply:
Now that would be a place where a comment would actually make sense.

Reply:
Most definitely. Comment should explain the unobvious "why".